### PR TITLE
Remove matlab scripts and revise documentations

### LINKED ISSUE #28: Revise readme
Readme is now used as "User guide" in the documentation. It should include:
- command line to install package
- simple example to use package
- links to notebooks

The section **Recipe of model execution** can be removed.

### LINKED ISSUE #31: Matlab R2021a  incompatible with Python 3.9
On Snellius and CRIB, matlab `R2021a` is installed that supports python 3.7 and 3.8. see [python compatibility](https://www.mathworks.com/content/dam/mathworks/mathworks-dot-com/support/sysreq/files/python-compatibility.pdf).

### LINKED ISSUE #33: README's matlab runtime installation instructions
In the README, linux users are instructed to unzip and then change the directory in the following way:
```bash
# Unzip the file
unzip MATLAB_Runtime_R2021a_Update_6_glnxa64.zip
# Install it
cd MATLAB_Runtime_R2021a_Update_6_glnxa64
```
However, the .zip file contains no main folder, but immediately has a lot of files in it. With `unzip` the destination directory is the current folder, which will then be filled with folders and files, and there will not be a `MATLAB_Runtime_R2021a_Update_6_glnxa64` folder.

Change:
`unzip MATLAB_Runtime_R2021a_Update_6_glnxa64.zip`
To:
`unzip MATLAB_Runtime_R2021a_Update_6_glnxa64.zip -d MATLAB_Runtime`

### PR DESCRIPTION
**Pull request by SarahAlidoost**
see https://github.com/EcoExtreML/STEMMUS_SCOPE/pull/111

closes #31 
closes #28 
closes #33

see also https://github.com/gaurav-nelson/github-action-markdown-link-check#disable-check-for-some-links

### CODE REVIEW

**Review by BSchilperoort — APPROVED** [review]
No big comments, except the python version issue :)

File: environment.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+name: pystemmusscope
 channels:
   - conda-forge
 dependencies:
-  - python=3.9
-  - netcdf4>=1.5.8,<2
-  - pandas
-  - jupyterlab
+  - pip
+  - python=3.8 # supported by matlab R2021a, see issue #31
```
[the remark this answers is not in the document]
Comment (SarahAlidoost) [reply environment.yml]: see my reply on #31 and the new changes to this PR.

File: README.md [hunk 2]
```diff
@@ -73,16 +64,16 @@ In a terminal:
...
+cd MATLAB_Runtime
 sudo -H ./install -mode silent -agreeToLicense yes
 ```
 
-For more information on how to download and install it, see the links below:
-- [download](https://nl.mathworks.com/products/compiler/matlab-runtime.html)
-- [intallation](https://nl.mathworks.com/help/compiler/install-the-matlab-runtime.html)
+For more information on how to download and install MATLAB Runtime, see the links below:
+  - [download](https://nl.mathworks.com/products/compiler/matlab-runtime.html)
+  - [intallation](https://nl.mathworks.com/help/compiler/install-the-matlab-runtime.html)
```
Comment (BSchilperoort) [inline README.md:76]:
```suggestion
  - [installation](https://nl.mathworks.com/help/compiler/install-the-matlab-runtime.html)
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -65,32 +65,80 @@ isort
 ## Development of STEMMUS_SCOPE model
 
 <!-- markdown-link-check-disable-next-line -->
-To contribuute to STEMMUS_SCOPE model, you need access to model source code that is stored in the repository [STEMMUS_SCOPE](https://github.com/EcoExtreML/STEMMUS_SCOPE). You also need a MATLAB license.
+To contribute to STEMMUS_SCOPE model, you need access to model source code that is stored in the repository [STEMMUS_SCOPE](https://github.com/EcoExtreML/STEMMUS_SCOPE). You also need a MATLAB license.
```
Comment (BSchilperoort) [inline CONTRIBUTING.md:68]:
```suggestion
To contribute to the STEMMUS_SCOPE model, you need access to the model source code that is stored in the repository [STEMMUS_SCOPE](https://github.com/EcoExtreML/STEMMUS_SCOPE). You also need a MATLAB license.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -65,32 +65,80 @@ isort
...
+
+```
+</details>
+
 
 ### Development on CRIB using MATLAB
 
 [CRIB](https://crib.utwente.nl/) is the ITC Geospatial Computing Platform.
 
+MATLAB `2021a` is installed on CRIB that is compatiable with Python `3.8`, see [Versions of Python Compatible with MATLAB Products](https://www.mathworks.com/content/dam/mathworks/mathworks-dot-com/support/sysreq/files/python-compatibility.pdf).
```
Comment (BSchilperoort) [inline CONTRIBUTING.md:112]:
```suggestion
MATLAB `2021a` is installed on CRIB, and supports Python `3.8`, see [Versions of Python Compatible with MATLAB Products](https://www.mathworks.com/content/dam/mathworks/mathworks-dot-com/support/sysreq/files/python-compatibility.pdf).
```

**Review by BSchilperoort — APPROVED** [review]
Nice work, Sarah. I only have some small textual comments.

It's much better this way, only having CRIB users to use Python 3.8 👍

**Pull request merged**